SparkPost/2web2ui
# AC-646: Hid subaccount section if subaccount user

### What Changed
 - Templates hides field for subaccount users on template edit page
-  Snippets hides field for subaccount users on snippet edit page

### How To Test
 - To Add subaccount users:
    - [Use this link](https://github.com/SparkPost/2web2ui/blob/master/docs/feature-flags.md#give-access-to-your-feature) to add the ui option flag "subaccount_reporting"
    - Use an admin account to create a subaccount
    - Use the admin account to invite a subaccount reporting user with "Manage Users" -> "Invite User"
      - Invite email doesn't work on appteam because SSO so I recommend creating an account
 - Check that the fields are absent on subaccount users
 - Check that the fields are present on non subaccount users
 - Check that form submission works for users with edit permissions

### To Do
